calculator.js

```diff
@@ -23,6 +13,73 @@ const calculateUserInput = function (error, promptInput) {
 //   operation: 'add',
 // });
 
+function getValidOperator(operator) {
+const operators = ['+', 'add', '-', 'subtract', '*', 'multiply', '/', 'divide']; 
+
+  if (operators.includes(operator)) {
+    return operator;
+  }
+  return undefined;
+}
+
+function getValidNumber(number) {
+    return parseInt(number);
+}
+
+const calculateUserInput = function (error, promptInput) {
+  // console.log('This is the value of the promptInput variable that got passed in by prompt, after our prompt package collect user input', promptInput);
+
+  let operator = getValidOperator(promptInput.operation);
+  if (operator == null) {
+    console.log(`unsupported operator`);
+    return;
+  }
+
+
+  let num1 = getValidNumber(promptInput.num1);
+  if (isNaN(num1)) {
+    console.log(`${promptInput.num1} is not a number.`);
+    return;
+  }
```

Comment:
I'd like to see the check for if the number `isNaN` moved into `getValidNumber`.